(gimp) File download location has changed

### Chocolatey Version

Other (note in the comments)

### Chocolatey License

_No response_

### Package Version

2.10.24 upgrade to 2.10.30 Windows

### Current Behaviour

Chocolatey Version 1.1.0
Upgrade failed with message "The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://download.gimp.org/mirror/pub/gimp/v2.10/windows/gimp-2.10.30-setup.exe'".

So I went and checked the gimp download website and the file path has changed. It no longer uses the mirror subdirectory. the correct path for the install.exe is "https://download.gimp.org/pub/gimp/v2.10/windows/". Would it be possible to correct this please.

### Expected Behaviour

New path to install executable should be
"https://download.gimp.org/pub/gimp/v2.10/windows/gimp-2.10.30-setup.exe"

Thanks!

### Steps To Reproduce

1. Chocolatey v1.1.0
2. Windows powershell
3. choco upgrade gimp

### Environment

```markdown
- Operating System:Microsoft Windows NT 10.0.19043.0
- PowerShell Version: 5.1.19041.1320
- Shell: Administrator: Windows PowerShell
```


### Chocolatey Log

```shell
2022-04-12 19:20:24,016 8076 [DEBUG] - Command line: "C:\ProgramData\chocolatey\choco.exe" upgrade gimp -y
2022-04-12 19:20:24,016 8076 [DEBUG] - Received arguments: upgrade gimp -y
2022-04-12 19:20:24,048 8076 [DEBUG] - RemovePendingPackagesTask is now ready and waiting for PreRunMessage.
2022-04-12 19:20:24,048 8076 [DEBUG] - Sending message 'PreRunMessage' out if there are subscribers...
2022-04-12 19:20:24,048 8076 [DEBUG] - [Pending] Removing all pending packages that should not be considered installed...
2022-04-12 19:20:24,095 8076 [DEBUG] - Performing validation checks.
2022-04-12 19:20:24,095 8076 [DEBUG] - Global Configuration Validation Checks:
2022-04-12 19:20:24,095 8076 [DEBUG] -  - Package Exit Code / Exit On Reboot = Checked
2022-04-12 19:20:24,095 8076 [DEBUG] - System State Validation Checks:
2022-04-12 19:20:24,095 8076 [DEBUG] -  Reboot Requirement Checks:
2022-04-12 19:20:24,095 8076 [DEBUG] -  - Pending Computer Rename = Checked
2022-04-12 19:20:24,095 8076 [DEBUG] -  - Pending Component Based Servicing = Checked
2022-04-12 19:20:24,095 8076 [DEBUG] -  - Pending Windows Auto Update = Checked
2022-04-12 19:20:24,095 8076 [DEBUG] -  - Pending File Rename Operations = Checked
2022-04-12 19:20:24,095 8076 [DEBUG] -  - Pending Windows Package Installer = Checked
2022-04-12 19:20:24,095 8076 [DEBUG] -  - Pending Windows Package Installer SysWow64 = Checked
2022-04-12 19:20:24,095 8076 [INFO ] - 2 validations performed. 2 success(es), 0 warning(s), and 0 error(s).
2022-04-12 19:20:24,110 8076 [DEBUG] - The source 'https://chocolatey.org/api/v2/' evaluated to a 'normal' source type
2022-04-12 19:20:24,110 8076 [DEBUG] - 
NOTE: Hiding sensitive configuration data! Please double and triple
 check to be sure no sensitive data is shown, especially if copying
 output to a gist for review.
2022-04-12 19:20:24,110 8076 [DEBUG] - Configuration: CommandName='upgrade'|
CacheLocation='C:\Users\dplhu\AppData\Local\Temp\chocolatey'|
ContainsLegacyPackageInstalls='True'|
CommandExecutionTimeoutSeconds='2700'|WebRequestTimeoutSeconds='30'|
Sources='https://chocolatey.org/api/v2/'|SourceType='normal'|
Debug='False'|Verbose='False'|Trace='False'|Force='False'|Noop='False'|
HelpRequested='False'|UnsuccessfulParsing='False'|RegularOutput='True'|
QuietOutput='False'|PromptForConfirmation='False'|
DisableCompatibilityChecks='False'|AcceptLicense='True'|
AllowUnofficialBuild='False'|Input='gimp'|AllVersions='False'|
SkipPackageInstallProvider='False'|PackageNames='gimp'|
Prerelease='False'|ForceX86='False'|OverrideArguments='False'|
NotSilent='False'|ApplyPackageParametersToDependencies='False'|
ApplyInstallArgumentsToDependencies='False'|IgnoreDependencies='False'|
AllowMultipleVersions='False'|AllowDowngrade='False'|
ForceDependencies='False'|Information.PlatformType='Windows'|
Information.PlatformVersion='10.0.19043.0'|
Information.PlatformName='Windows 10'|
Information.ChocolateyVersion='1.1.0.0'|
Information.ChocolateyProductVersion='1.1.0'|
Information.FullName='choco, Version=1.1.0.0, Culture=neutral, PublicKeyToken=79d02ea9cad655eb'|

Information.Is64BitOperatingSystem='True'|
Information.Is64BitProcess='True'|Information.IsInteractive='True'|
Information.UserName='dplhu'|Information.UserDomainName='DOUGLAPTOPAS'|
Information.IsUserAdministrator='True'|
Information.IsUserSystemAccount='False'|
Information.IsUserRemoteDesktop='False'|
Information.IsUserRemote='True'|
Information.IsProcessElevated='True'|
Information.IsLicensedVersion='False'|Information.LicenseType='Foss'|
Information.CurrentDirectory='C:\'|Features.AutoUninstaller='True'|
Features.ChecksumFiles='True'|Features.AllowEmptyChecksums='False'|
Features.AllowEmptyChecksumsSecure='True'|
Features.FailOnAutoUninstaller='False'|
Features.FailOnStandardError='False'|Features.UsePowerShellHost='True'|
Features.LogEnvironmentValues='False'|Features.LogWithoutColor='False'|
Features.VirusCheck='False'|
Features.FailOnInvalidOrMissingLicense='False'|
Features.IgnoreInvalidOptionsSwitches='True'|
Features.UsePackageExitCodes='True'|
Features.UseEnhancedExitCodes='False'|
Features.UseFipsCompliantChecksums='False'|
Features.ShowNonElevatedWarnings='True'|
Features.ShowDownloadProgress='True'|
Features.StopOnFirstPackageFailure='False'|
Features.UseRememberedArgumentsForUpgrades='True'|
Features.IgnoreUnfoundPackagesOnUpgradeOutdated='False'|
Features.SkipPackageUpgradesWhenNotInstalled='False'|
Features.RemovePackageInformationOnUninstall='False'|
Features.ExitOnRebootDetected='False'|
Features.LogValidationResultsOnWarnings='True'|
Features.UsePackageRepositoryOptimizations='True'|
ListCommand.LocalOnly='False'|ListCommand.IdOnly='False'|
ListCommand.IncludeRegistryPrograms='False'|ListCommand.PageSize='25'|
ListCommand.Exact='False'|ListCommand.ByIdOnly='False'|
ListCommand.ByTagOnly='False'|ListCommand.IdStartsWith='False'|
ListCommand.OrderByPopularity='False'|ListCommand.ApprovedOnly='False'|
ListCommand.DownloadCacheAvailable='False'|
ListCommand.NotBroken='False'|
ListCommand.IncludeVersionOverrides='False'|
UpgradeCommand.FailOnUnfound='False'|
UpgradeCommand.FailOnNotInstalled='False'|
UpgradeCommand.NotifyOnlyAvailableUpgrades='False'|
UpgradeCommand.ExcludePrerelease='False'|
NewCommand.AutomaticPackage='False'|
NewCommand.UseOriginalTemplate='False'|SourceCommand.Command='unknown'|
SourceCommand.Priority='0'|SourceCommand.BypassProxy='False'|
SourceCommand.AllowSelfService='False'|
SourceCommand.VisibleToAdminsOnly='False'|
FeatureCommand.Command='unknown'|ConfigCommand.Command='unknown'|
ApiKeyCommand.Remove='False'|PinCommand.Command='unknown'|
OutdatedCommand.IgnorePinned='False'|
ExportCommand.IncludeVersionNumbers='False'|Proxy.BypassOnLocal='True'|
TemplateCommand.Command='unknown'|
2022-04-12 19:20:24,110 8076 [DEBUG] - _ Chocolatey:ChocolateyUpgradeCommand - Normal Run Mode _
2022-04-12 19:20:24,110 8076 [INFO ] - Upgrading the following packages:
2022-04-12 19:20:24,126 8076 [INFO ] - gimp
2022-04-12 19:20:24,126 8076 [INFO ] - By upgrading, you accept licenses for the packages.
2022-04-12 19:20:24,266 8076 [DEBUG] - Using 'https://chocolatey.org/api/v2/'.
2022-04-12 19:21:48,917 8076 [DEBUG] - - Supports prereleases? 'True'.
2022-04-12 19:21:48,917 8076 [DEBUG] - - Is ServiceBased? 'True'.
2022-04-12 19:21:49,167 8076 [DEBUG] - Package 'gimp' found on source 'https://chocolatey.org/api/v2/'
2022-04-12 19:21:49,183 8076 [WARN ] - 
You have gimp v2.10.24.3 installed. Version 2.10.30 is available based on your source(s).
2022-04-12 19:21:49,199 8076 [DEBUG] - gimp - Adding remembered arguments for upgrade:  --cache-location="'C:\Users\dplhu\AppData\Local\Temp\chocolatey'"
2022-04-12 19:21:49,261 8076 [DEBUG] - Backing up existing gimp prior to operation.
2022-04-12 19:21:49,261 8076 [DEBUG] - Moving 'C:\ProgramData\chocolatey\lib\gimp'
 to 'C:\ProgramData\chocolatey\lib-bkp\gimp'
2022-04-12 19:21:51,275 8076 [DEBUG] - Attempting to create directory "C:\ProgramData\chocolatey\lib\gimp".
2022-04-12 19:21:51,275 8076 [DEBUG] - Attempting to copy "C:\ProgramData\chocolatey\lib-bkp\gimp\gimp.nupkg"
 to "C:\ProgramData\chocolatey\lib\gimp\gimp.nupkg".
2022-04-12 19:21:51,275 8076 [DEBUG] - Attempting to copy "C:\ProgramData\chocolatey\lib-bkp\gimp\gimp.nuspec"
 to "C:\ProgramData\chocolatey\lib\gimp\gimp.nuspec".
2022-04-12 19:21:51,275 8076 [DEBUG] - Attempting to create directory "C:\ProgramData\chocolatey\lib\gimp\tools".
2022-04-12 19:21:51,275 8076 [DEBUG] - Attempting to copy "C:\ProgramData\chocolatey\lib-bkp\gimp\tools\chocolateyInstall.ps1"
 to "C:\ProgramData\chocolatey\lib\gimp\tools\chocolateyInstall.ps1".
2022-04-12 19:21:52,797 8076 [DEBUG] - Deleting file C:\ProgramData\chocolatey\lib\gimp\tools\chocolateyInstall.ps1
2022-04-12 19:21:52,797 8076 [DEBUG] - Attempting to delete file "C:\ProgramData\chocolatey\lib\gimp\tools\chocolateyInstall.ps1".
2022-04-12 19:21:52,797 8076 [DEBUG] - Capturing package files in 'C:\ProgramData\chocolatey\lib\gimp'
2022-04-12 19:21:52,797 8076 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\gimp\gimp.nupkg'
  with checksum 'E1973718DB1E071BAD4208030A3B579E'
2022-04-12 19:21:52,797 8076 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\gimp\gimp.nuspec'
  with checksum '488718BD1CAF04C6918C92867BB4DCE8'
2022-04-12 19:22:36,697 8076 [INFO ] - [NuGet] Uninstalling 'gimp 2.10.24.3'.
2022-04-12 19:22:36,713 8076 [DEBUG] - [NuGet] Removed folder 'C:\ProgramData\chocolatey\lib\gimp\tools'.
2022-04-12 19:22:36,713 8076 [DEBUG] - [NuGet] Removed file 'gimp.nuspec' to folder 'C:\ProgramData\chocolatey\lib\gimp'.
2022-04-12 19:22:36,713 8076 [DEBUG] - [NuGet] Removed file 'gimp.nupkg' to folder 'C:\ProgramData\chocolatey\lib\gimp'.
2022-04-12 19:22:36,713 8076 [DEBUG] - [NuGet] Removed folder 'C:\ProgramData\chocolatey\lib\gimp'.
2022-04-12 19:22:36,713 8076 [INFO ] - [NuGet] Successfully uninstalled 'gimp 2.10.24.3'.
2022-04-12 19:22:36,729 8076 [INFO ] - [NuGet] Installing 'gimp 2.10.30'.
2022-04-12 19:22:36,729 8076 [DEBUG] - [NuGet] Added file 'chocolateyInstall.ps1' to folder 'gimp\tools'.
2022-04-12 19:22:36,729 8076 [DEBUG] - [NuGet] Added file 'gimp.nupkg' to folder 'gimp'.
2022-04-12 19:22:36,970 8076 [DEBUG] - [NuGet] Added file 'gimp.nuspec' to folder 'gimp'.
2022-04-12 19:22:36,970 8076 [INFO ] - [NuGet] Successfully installed 'gimp 2.10.30'.
2022-04-12 19:22:36,970 8076 [INFO ] - 
gimp v2.10.30 [Approved]
2022-04-12 19:22:36,985 8076 [INFO ] - gimp package files upgrade completed. Performing other installation steps.
2022-04-12 19:22:37,032 8076 [DEBUG] - Setting installer args for gimp
2022-04-12 19:22:37,032 8076 [DEBUG] - Setting package parameters for gimp
2022-04-12 19:22:37,032 8076 [DEBUG] - Contents of 'C:\ProgramData\chocolatey\lib\gimp\tools\chocolateyInstall.ps1':
2022-04-12 19:22:37,032 8076 [DEBUG] - $ErrorActionPreference = 'Stop'

$packageArgs = @{
  packageName    = $env:ChocolateyPackageName
  fileType       = 'exe'
  url            = 'https://download.gimp.org/mirror/pub/gimp/v2.10/windows/gimp-2.10.30-setup.exe'
  softwareName   = 'GIMP'
  checksum       = '5b8db574966d2427fa02202b591173905f64806559e37fbdca4654e55b568c8e'
  checksumType   = 'sha256'
  silentArgs     = "/VERYSILENT /NORESTART /RESTARTEXITCODE=3010 /SUPPRESSMSGBOXES /SP- /LOG=`"$($env:TEMP)\$($env:chocolateyPackageName).$($env:chocolateyPackageVersion).InnoInstall.log`""
  validExitCodes = @(0)
}

Install-ChocolateyPackage @packageArgs

2022-04-12 19:22:37,048 8076 [DEBUG] - Calling built-in PowerShell host with ['[System.Threading.Thread]::CurrentThread.CurrentCulture = '';[System.Threading.Thread]::CurrentThread.CurrentUICulture = ''; & import-module -name 'C:\ProgramData\chocolatey\helpers\chocolateyInstaller.psm1'; & 'C:\ProgramData\chocolatey\helpers\chocolateyScriptRunner.ps1' -packageScript 'C:\ProgramData\chocolatey\lib\gimp\tools\chocolateyInstall.ps1' -installArguments '' -packageParameters ''']
2022-04-12 19:22:37,048 8076 [DEBUG] - Redirecting System.Management.Automation.resources, Version=3.0.0.0, Culture=en-US, PublicKeyToken=31bf3856ad364e35, requested by ''
2022-04-12 19:22:37,204 8076 [DEBUG] - Host version is 5.1.19041.1, PowerShell Version is '5.1.19041.1320' and CLR Version is '4.0.30319.42000'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Format-FileSize'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Get-ChecksumValid'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Get-ChocolateyUnzip'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Get-ChocolateyWebFile'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Get-EnvironmentVariable'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Get-EnvironmentVariableNames'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Get-FtpFile'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Get-OSArchitectureWidth'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Get-PackageParameters'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Get-PackageParametersBuiltIn'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Get-ToolsLocation'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Get-UACEnabled'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Get-UninstallRegistryKey'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Get-VirusCheckValid'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Get-WebFile'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Get-WebFileName'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Get-WebHeaders'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Install-BinFile'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyEnvironmentVariable'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyExplorerMenuItem'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyFileAssociation'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyInstallPackage'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyPackage'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyPath'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyPinnedTaskBarItem'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyPowershellCommand'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyShortcut'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyVsixPackage'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyZipPackage'.
2022-04-12 19:22:37,657 8076 [INFO ] - VERBOSE: Exporting function 'Install-Vsix'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting function 'Set-EnvironmentVariable'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting function 'Set-PowerShellExitCode'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting function 'Start-ChocolateyProcessAsAdmin'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting function 'Test-ProcessAdminRights'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting function 'Uninstall-BinFile'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting function 'Uninstall-ChocolateyEnvironmentVariable'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting function 'Uninstall-ChocolateyPackage'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting function 'Uninstall-ChocolateyZipPackage'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting function 'Update-SessionEnvironment'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting function 'Write-FunctionCallLogMessage'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting alias 'Get-ProcessorBits'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting alias 'Get-OSBitness'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting alias 'Get-InstallRegistryKey'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting alias 'Generate-BinFile'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting alias 'Add-BinFile'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting alias 'Start-ChocolateyProcess'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting alias 'Invoke-ChocolateyProcess'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting alias 'Remove-BinFile'.
2022-04-12 19:22:37,673 8076 [INFO ] - VERBOSE: Exporting alias 'refreshenv'.
2022-04-12 19:22:37,673 8076 [DEBUG] - Loading community extensions
2022-04-12 19:22:37,688 8076 [DEBUG] - Importing 'C:\ProgramData\chocolatey\extensions\chocolatey-core\chocolatey-core.psm1'
2022-04-12 19:22:37,688 8076 [INFO ] - VERBOSE: Loading module from path 'C:\ProgramData\chocolatey\extensions\chocolatey-core\chocolatey-core.psm1'.
2022-04-12 19:22:37,782 8076 [INFO ] - VERBOSE: Exporting function 'Get-UninstallRegistryKey'.
2022-04-12 19:22:37,782 8076 [INFO ] - VERBOSE: Exporting function 'Get-AppInstallLocation'.
2022-04-12 19:22:37,782 8076 [INFO ] - VERBOSE: Exporting function 'Get-AvailableDriveLetter'.
2022-04-12 19:22:37,782 8076 [INFO ] - VERBOSE: Exporting function 'Get-EffectiveProxy'.
2022-04-12 19:22:37,782 8076 [INFO ] - VERBOSE: Exporting function 'Get-PackageCacheLocation'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Exporting function 'Get-PackageParameters'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Exporting function 'Get-WebContent'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Exporting function 'Register-Application'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Exporting function 'Remove-Process'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Importing function 'Get-AppInstallLocation'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Importing function 'Get-AvailableDriveLetter'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Importing function 'Get-EffectiveProxy'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Importing function 'Get-PackageCacheLocation'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Importing function 'Get-PackageParameters'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Importing function 'Get-UninstallRegistryKey'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Importing function 'Get-WebContent'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Importing function 'Register-Application'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Importing function 'Remove-Process'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Exporting function 'Format-FileSize'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Exporting function 'Get-ChecksumValid'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Exporting function 'Get-ChocolateyUnzip'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Exporting function 'Get-ChocolateyWebFile'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Exporting function 'Get-EnvironmentVariable'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Exporting function 'Get-EnvironmentVariableNames'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Exporting function 'Get-FtpFile'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Exporting function 'Get-OSArchitectureWidth'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Exporting function 'Get-PackageParameters'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Exporting function 'Get-PackageParametersBuiltIn'.
2022-04-12 19:22:37,798 8076 [INFO ] - VERBOSE: Exporting function 'Get-ToolsLocation'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Get-UACEnabled'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Get-UninstallRegistryKey'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Get-VirusCheckValid'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Get-WebFile'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Get-WebFileName'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Get-WebHeaders'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Install-BinFile'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyEnvironmentVariable'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyExplorerMenuItem'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyFileAssociation'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyInstallPackage'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyPackage'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyPath'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyPinnedTaskBarItem'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyPowershellCommand'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyShortcut'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyVsixPackage'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Install-ChocolateyZipPackage'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Install-Vsix'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Set-EnvironmentVariable'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Set-PowerShellExitCode'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Start-ChocolateyProcessAsAdmin'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Test-ProcessAdminRights'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Uninstall-BinFile'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Uninstall-ChocolateyEnvironmentVariable'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Uninstall-ChocolateyPackage'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Uninstall-ChocolateyZipPackage'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Update-SessionEnvironment'.
2022-04-12 19:22:37,813 8076 [INFO ] - VERBOSE: Exporting function 'Write-FunctionCallLogMessage'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Exporting function 'Get-AppInstallLocation'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Exporting function 'Get-AvailableDriveLetter'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Exporting function 'Get-EffectiveProxy'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Exporting function 'Get-PackageCacheLocation'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Exporting function 'Get-WebContent'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Exporting function 'Register-Application'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Exporting function 'Remove-Process'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Exporting alias 'Get-ProcessorBits'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Exporting alias 'Get-OSBitness'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Exporting alias 'Get-InstallRegistryKey'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Exporting alias 'Generate-BinFile'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Exporting alias 'Add-BinFile'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Exporting alias 'Start-ChocolateyProcess'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Exporting alias 'Invoke-ChocolateyProcess'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Exporting alias 'Remove-BinFile'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Exporting alias 'refreshenv'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Importing function 'Format-FileSize'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Importing function 'Get-AppInstallLocation'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Importing function 'Get-AvailableDriveLetter'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Importing function 'Get-ChecksumValid'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Importing function 'Get-ChocolateyUnzip'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Importing function 'Get-ChocolateyWebFile'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Importing function 'Get-EffectiveProxy'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Importing function 'Get-EnvironmentVariable'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Importing function 'Get-EnvironmentVariableNames'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Importing function 'Get-FtpFile'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Importing function 'Get-OSArchitectureWidth'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Importing function 'Get-PackageCacheLocation'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Importing function 'Get-PackageParameters'.
2022-04-12 19:22:37,829 8076 [INFO ] - VERBOSE: Importing function 'Get-PackageParametersBuiltIn'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Get-ToolsLocation'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Get-UACEnabled'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Get-UninstallRegistryKey'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Get-VirusCheckValid'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Get-WebContent'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Get-WebFile'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Get-WebFileName'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Get-WebHeaders'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Install-BinFile'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Install-ChocolateyEnvironmentVariable'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Install-ChocolateyExplorerMenuItem'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Install-ChocolateyFileAssociation'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Install-ChocolateyInstallPackage'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Install-ChocolateyPackage'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Install-ChocolateyPath'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Install-ChocolateyPinnedTaskBarItem'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Install-ChocolateyPowershellCommand'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Install-ChocolateyShortcut'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Install-ChocolateyVsixPackage'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Install-ChocolateyZipPackage'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Install-Vsix'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Register-Application'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Remove-Process'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Set-EnvironmentVariable'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Set-PowerShellExitCode'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Start-ChocolateyProcessAsAdmin'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Test-ProcessAdminRights'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Uninstall-BinFile'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Uninstall-ChocolateyEnvironmentVariable'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Uninstall-ChocolateyPackage'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Uninstall-ChocolateyZipPackage'.
2022-04-12 19:22:37,845 8076 [INFO ] - VERBOSE: Importing function 'Update-SessionEnvironment'.
2022-04-12 19:22:37,860 8076 [INFO ] - VERBOSE: Importing function 'Write-FunctionCallLogMessage'.
2022-04-12 19:22:37,860 8076 [INFO ] - VERBOSE: Importing alias 'Add-BinFile'.
2022-04-12 19:22:37,860 8076 [INFO ] - VERBOSE: Importing alias 'Generate-BinFile'.
2022-04-12 19:22:37,860 8076 [INFO ] - VERBOSE: Importing alias 'Get-InstallRegistryKey'.
2022-04-12 19:22:37,860 8076 [INFO ] - VERBOSE: Importing alias 'Get-OSBitness'.
2022-04-12 19:22:37,860 8076 [INFO ] - VERBOSE: Importing alias 'Get-ProcessorBits'.
2022-04-12 19:22:37,860 8076 [INFO ] - VERBOSE: Importing alias 'Invoke-ChocolateyProcess'.
2022-04-12 19:22:37,860 8076 [INFO ] - VERBOSE: Importing alias 'refreshenv'.
2022-04-12 19:22:37,860 8076 [INFO ] - VERBOSE: Importing alias 'Remove-BinFile'.
2022-04-12 19:22:37,860 8076 [INFO ] - VERBOSE: Importing alias 'Start-ChocolateyProcess'.
2022-04-12 19:22:37,876 8076 [DEBUG] - ---------------------------Script Execution---------------------------
2022-04-12 19:22:37,876 8076 [DEBUG] - Running 'ChocolateyScriptRunner' for gimp v2.10.30 with packageScript 'C:\ProgramData\chocolatey\lib\gimp\tools\chocolateyInstall.ps1', packageFolder:'C:\ProgramData\chocolatey\lib\gimp', installArguments: '', packageParameters: '',
2022-04-12 19:22:37,891 8076 [DEBUG] - Running 'C:\ProgramData\chocolatey\lib\gimp\tools\chocolateyInstall.ps1'
2022-04-12 19:22:37,923 8076 [DEBUG] - Running Install-ChocolateyPackage -silentArgs '/VERYSILENT /NORESTART /RESTARTEXITCODE=3010 /SUPPRESSMSGBOXES /SP- /LOG="C:\Users\dplhu\AppData\Local\Temp\chocolatey\gimp.2.10.30.InnoInstall.log"' -checksumType 'sha256' -url 'https://download.gimp.org/mirror/pub/gimp/v2.10/windows/gimp-2.10.30-setup.exe' -checksum '5b8db574966d2427fa02202b591173905f64806559e37fbdca4654e55b568c8e' -packageName 'gimp' -validExitCodes '0' -fileType 'exe' 
2022-04-12 19:22:37,954 8076 [DEBUG] - Running Get-ChocolateyWebFile -packageName 'gimp' -fileFullPath 'C:\Users\dplhu\AppData\Local\Temp\chocolatey\gimp\2.10.30\gimpInstall.exe' -url 'https://download.gimp.org/mirror/pub/gimp/v2.10/windows/gimp-2.10.30-setup.exe' -url64bit '' -checksum '5b8db574966d2427fa02202b591173905f64806559e37fbdca4654e55b568c8e' -checksumType 'sha256' -checksum64 '' -checksumType64 '' -options 'System.Collections.Hashtable' -getOriginalFileName 'True' 
2022-04-12 19:22:37,970 8076 [DEBUG] - Running Get-ProcessorBits -compare '64' 
2022-04-12 19:22:37,970 8076 [DEBUG] - CPU is 64 bit
2022-04-12 19:22:38,001 8076 [DEBUG] - Running Get-WebFileName -url 'https://download.gimp.org/mirror/pub/gimp/v2.10/windows/gimp-2.10.30-setup.exe' -defaultName 'gimpInstall.exe' 
2022-04-12 19:23:02,034 8076 [DEBUG] - Using response url to determine file name. 'https://mirror.kumi.systems/gimp/pub/gimp/v2.10/windows/gimp-2.10.30-setup.exe'
2022-04-12 19:23:02,034 8076 [DEBUG] - File name determined from url is 'gimp-2.10.30-setup.exe'
2022-04-12 19:23:02,049 8076 [DEBUG] - Running Get-WebHeaders -url 'https://download.gimp.org/mirror/pub/gimp/v2.10/windows/gimp-2.10.30-setup.exe' -ErrorAction 'Stop' 
2022-04-12 19:23:02,049 8076 [DEBUG] - Setting the UserAgent to 'chocolatey command line'
2022-04-12 19:23:02,049 8076 [DEBUG] - Request Headers:
2022-04-12 19:23:02,049 8076 [DEBUG] -   'Accept':'*/*'
2022-04-12 19:23:02,049 8076 [DEBUG] -   'User-Agent':'chocolatey command line'
2022-04-12 19:23:33,135 8076 [INFO ] - Attempt to get headers for https://download.gimp.org/mirror/pub/gimp/v2.10/windows/gimp-2.10.30-setup.exe failed.
  The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://download.gimp.org/mirror/pub/gimp/v2.10/windows/gimp-2.10.30-setup.exe'. Exception calling "GetResponse" with "0" argument(s): "The operation has timed out"
2022-04-12 19:23:33,135 8076 [INFO ] - Downloading gimp 
  from 'https://download.gimp.org/mirror/pub/gimp/v2.10/windows/gimp-2.10.30-setup.exe'
2022-04-12 19:23:33,150 8076 [DEBUG] - Running Get-WebFile -url 'https://download.gimp.org/mirror/pub/gimp/v2.10/windows/gimp-2.10.30-setup.exe' -fileName 'C:\Users\dplhu\AppData\Local\Temp\chocolatey\gimp\2.10.30\gimp-2.10.30-setup.exe' -options 'System.Collections.Hashtable' 
2022-04-12 19:23:33,150 8076 [DEBUG] - Setting request timeout to  30000
2022-04-12 19:23:33,166 8076 [DEBUG] - Setting read/write timeout to  2700000
2022-04-12 19:23:33,166 8076 [DEBUG] - Setting the UserAgent to 'chocolatey command line'
2022-04-12 19:24:05,237 8076 [ERROR] - ERROR: The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://download.gimp.org/mirror/pub/gimp/v2.10/windows/gimp-2.10.30-setup.exe'. Exception calling "GetResponse" with "0" argument(s): "The operation has timed out" 
This package is likely not broken for licensed users - see https://docs.chocolatey.org/en-us/features/private-cdn.
2022-04-12 19:24:05,237 8076 [DEBUG] - Built-in PowerShell host called with ['[System.Threading.Thread]::CurrentThread.CurrentCulture = '';[System.Threading.Thread]::CurrentThread.CurrentUICulture = ''; & import-module -name 'C:\ProgramData\chocolatey\helpers\chocolateyInstaller.psm1'; & 'C:\ProgramData\chocolatey\helpers\chocolateyScriptRunner.ps1' -packageScript 'C:\ProgramData\chocolatey\lib\gimp\tools\chocolateyInstall.ps1' -installArguments '' -packageParameters '''] exited with '404'.
2022-04-12 19:24:05,252 8076 [DEBUG] - Calling command ['"C:\WINDOWS\System32\shutdown.exe" /a']
2022-04-12 19:24:05,283 8076 [DEBUG] - Command ['"C:\WINDOWS\System32\shutdown.exe" /a'] exited with '1116'
2022-04-12 19:24:05,314 8076 [DEBUG] - Capturing package files in 'C:\ProgramData\chocolatey\lib\gimp'
2022-04-12 19:24:05,320 8076 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\gimp\gimp.nupkg'
  with checksum '81C6E1FAC81DE57EE81C2DBD715EE462'
2022-04-12 19:24:05,320 8076 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\gimp\gimp.nuspec'
  with checksum 'F75EDF49298522F77AFAB605195134DE'
2022-04-12 19:24:05,320 8076 [DEBUG] -  Found 'C:\ProgramData\chocolatey\lib\gimp\tools\chocolateyInstall.ps1'
  with checksum '327480D041A1ECC7117EEE6127EDCA70'
2022-04-12 19:24:05,320 8076 [DEBUG] - Attempting to create directory "C:\ProgramData\chocolatey\.chocolatey\gimp.2.10.30".
2022-04-12 19:24:05,320 8076 [DEBUG] - There was no original file at 'C:\ProgramData\chocolatey\.chocolatey\gimp.2.10.30\.files'
2022-04-12 19:24:05,320 8076 [DEBUG] - Attempting to delete file "C:\ProgramData\chocolatey\.chocolatey\gimp.2.10.30\.arguments".
2022-04-12 19:24:05,320 8076 [DEBUG] - Attempting to delete file "C:\ProgramData\chocolatey\.chocolatey\gimp.2.10.30\.extra".
2022-04-12 19:24:05,335 8076 [DEBUG] - Attempting to delete file "C:\ProgramData\chocolatey\.chocolatey\gimp.2.10.30\.version".
2022-04-12 19:24:05,335 8076 [DEBUG] - Attempting to delete file "C:\ProgramData\chocolatey\.chocolatey\gimp.2.10.30\.sxs".
2022-04-12 19:24:05,335 8076 [DEBUG] - Attempting to delete file "C:\ProgramData\chocolatey\.chocolatey\gimp.2.10.30\.pin".
2022-04-12 19:24:05,335 8076 [DEBUG] - Sending message 'HandlePackageResultCompletedMessage' out if there are subscribers...
2022-04-12 19:24:05,335 8076 [ERROR] - The upgrade of gimp was NOT successful.
2022-04-12 19:24:05,339 8076 [ERROR] - Error while running 'C:\ProgramData\chocolatey\lib\gimp\tools\chocolateyInstall.ps1'.
 See log for details.
2022-04-12 19:24:05,339 8076 [DEBUG] - Attempting to create directory "C:\ProgramData\chocolatey\lib-bad".
2022-04-12 19:24:05,339 8076 [DEBUG] - Moving 'C:\ProgramData\chocolatey\lib\gimp'
 to 'C:\ProgramData\chocolatey\lib-bad\gimp'
2022-04-12 19:24:07,364 8076 [DEBUG] - Attempting rollback
2022-04-12 19:24:07,364 8076 [DEBUG] - Moving 'C:\ProgramData\chocolatey\lib-bkp\gimp'
 to 'C:\ProgramData\chocolatey\lib\gimp'
2022-04-12 19:24:09,373 8076 [DEBUG] - Attempting to delete file "C:\Users\dplhu\AppData\Local\NuGet\Cache\gimp.2.10.30.nupkg".
2022-04-12 19:24:09,373 8076 [WARN ] - 
Chocolatey upgraded 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
2022-04-12 19:24:09,373 8076 [INFO ] - 
2022-04-12 19:24:09,388 8076 [ERROR] - Failures
2022-04-12 19:24:09,388 8076 [ERROR] -  - gimp (exited 404) - Error while running 'C:\ProgramData\chocolatey\lib\gimp\tools\chocolateyInstall.ps1'.
 See log for details.
2022-04-12 19:24:09,388 8076 [WARN ] - 
Did you know the proceeds of Pro (and some proceeds from other
 licensed editions) go into bettering the community infrastructure?
 Your support ensures an active community, keeps Chocolatey tip-top,
 plus it nets you some awesome features!
 https://chocolatey.org/compare
2022-04-12 19:24:09,388 8076 [DEBUG] - Sending message 'PostRunMessage' out if there are subscribers...
2022-04-12 19:24:09,404 8076 [DEBUG] - Exiting with 404
2022-04-12 19:59:15,034 5356 [DEBUG] - XmlConfiguration is now operational
2022-04-12 19:59:15,065 5356 [INFO ] - ============================================================
2022-04-12 19:59:15,416 5356 [INFO ] - Chocolatey v1.1.0
2022-04-12 19:59:15,424 5356 [DEBUG] - Chocolatey is running on Windows v 10.0.19043.0
2022-04-12 19:59:15,426 5356 [DEBUG] - Attempting to delete file "C:/ProgramData/chocolatey/choco.exe.old".
2022-04-12 19:59:15,427 5356 [DEBUG] - Attempting to delete file "C:\ProgramData\chocolatey\choco.exe.old".
2022-04-12 19:59:15,433 5356 [DEBUG] - Command line: "C:\ProgramData\chocolatey\choco.exe" outdated -v
2022-04-12 19:59:15,434 5356 [DEBUG] - Received arguments: outdated -v
2022-04-12 19:59:15,466 5356 [DEBUG] - RemovePendingPackagesTask is now ready and waiting for PreRunMessage.
2022-04-12 19:59:15,471 5356 [DEBUG] - Sending message 'PreRunMessage' out if there are subscribers...
2022-04-12 19:59:15,475 5356 [DEBUG] - [Pending] Removing all pending packages that should not be considered installed...
2022-04-12 19:59:15,505 5356 [DEBUG] - Performing validation checks.
2022-04-12 19:59:15,508 5356 [DEBUG] - Global Configuration Validation Checks:
2022-04-12 19:59:15,509 5356 [DEBUG] -  - Package Exit Code / Exit On Reboot = Checked
2022-04-12 19:59:15,510 5356 [DEBUG] - System State Validation Checks:
2022-04-12 19:59:15,511 5356 [DEBUG] -  Reboot Requirement Checks:
2022-04-12 19:59:15,513 5356 [DEBUG] -  - Pending Computer Rename = Checked
2022-04-12 19:59:15,514 5356 [DEBUG] -  - Pending Component Based Servicing = Checked
2022-04-12 19:59:15,515 5356 [DEBUG] -  - Pending Windows Auto Update = Checked
2022-04-12 19:59:15,515 5356 [DEBUG] -  - Pending File Rename Operations = Checked
2022-04-12 19:59:15,516 5356 [DEBUG] -  - Pending Windows Package Installer = Checked
2022-04-12 19:59:15,517 5356 [DEBUG] -  - Pending Windows Package Installer SysWow64 = Checked
2022-04-12 19:59:15,520 5356 [INFO ] - 2 validations performed. 2 success(es), 0 warning(s), and 0 error(s).
2022-04-12 19:59:15,521 5356 [DEBUG] - The source 'https://chocolatey.org/api/v2/' evaluated to a 'normal' source type
2022-04-12 19:59:15,522 5356 [DEBUG] - 
NOTE: Hiding sensitive configuration data! Please double and triple
 check to be sure no sensitive data is shown, especially if copying
 output to a gist for review.
2022-04-12 19:59:15,529 5356 [DEBUG] - Configuration: CommandName='outdated'|
CacheLocation='C:\Users\dplhu\AppData\Local\Temp\chocolatey'|
ContainsLegacyPackageInstalls='True'|
CommandExecutionTimeoutSeconds='2700'|WebRequestTimeoutSeconds='30'|
Sources='https://chocolatey.org/api/v2/'|SourceType='normal'|
Debug='False'|Verbose='True'|Trace='False'|Force='False'|Noop='False'|
HelpRequested='False'|UnsuccessfulParsing='False'|RegularOutput='True'|
QuietOutput='False'|PromptForConfirmation='True'|
DisableCompatibilityChecks='False'|AcceptLicense='False'|
AllowUnofficialBuild='False'|AllVersions='False'|
SkipPackageInstallProvider='False'|Prerelease='False'|ForceX86='False'|
OverrideArguments='False'|NotSilent='False'|
ApplyPackageParametersToDependencies='False'|
ApplyInstallArgumentsToDependencies='False'|IgnoreDependencies='False'|
AllowMultipleVersions='False'|AllowDowngrade='False'|
ForceDependencies='False'|Information.PlatformType='Windows'|
Information.PlatformVersion='10.0.19043.0'|
Information.PlatformName='Windows 10'|
Information.ChocolateyVersion='1.1.0.0'|
Information.ChocolateyProductVersion='1.1.0'|
Information.FullName='choco, Version=1.1.0.0, Culture=neutral, PublicKeyToken=79d02ea9cad655eb'|
```


### Anything else?

_No response_

--- a/automatic/gimp/tools/chocolateyInstall.ps1
+++ b/automatic/gimp/tools/chocolateyInstall.ps1
@@ -1,5 +1,7 @@
 ﻿$ErrorActionPreference = 'Stop'
 
+$fallbackUrl32 = 'https://download.gimp.org/pub/gimp/v2.10/windows/gimp-2.10.30-setup.exe'
+
 $packageArgs = @{
   packageName    = $env:ChocolateyPackageName
   fileType       = 'exe'
@@ -9,6 +11,13 @@ $packageArgs = @{
   checksumType   = 'sha256'
   silentArgs     = "/VERYSILENT /NORESTART /RESTARTEXITCODE=3010 /SUPPRESSMSGBOXES /SP- /LOG=`"$($env:TEMP)\$($env:chocolateyPackageName).$($env:chocolateyPackageVersion).InnoInstall.log`""
   validExitCodes = @(0)
+}
+
+Try {
+    Get-WebHeaders -Url $packageArgs.url
+} Catch {
+    Write-Warning "The mirror URL is not available, falling back to the main site"
+    $packageArgs.url = $fallbackUrl32
 }
 
 Install-ChocolateyPackage @packageArgs

--- a/automatic/gimp/update.ps1
+++ b/automatic/gimp/update.ps1
@@ -8,6 +8,7 @@ function global:au_SearchReplace {
       "(?i)^(\s*url\s*=\s*)'.*'" = "`${1}'$($Latest.URL32)'"
       "(?i)^(\s*checksum\s*=\s*)'.*'" = "`${1}'$($Latest.Checksum32)'"
       "(?i)^(\s*checksumType\s*=\s*)'.*'" = "`${1}'$($Latest.ChecksumType32)'"
+      "(^[$]fallbackUrl32\s*=\s*)('.*')" = "`$1'$($Latest.FallbackURL32)'"  
     }
   }
 }
@@ -17,12 +18,15 @@ function global:au_GetLatest {
 
   $re        = '\.exe$'
   $url32     = $download_page.Links | ? href -match $re | select -first 1 -expand href | % { 'https:' + $_ }
+  
+  $fallbackUrl32 = $url32 -replace "download.gimp.org/mirror", "download.gimp.org"
 
   $regex = "(?:[putesmigx]+)\-|\.exe"; $regex01 = "(\-)"; $regex02 = "RC"
   $version32 = ((($url32 -split("\/"))[-1]) -replace($regex,"") )
   if ($version32 -notmatch $regex02 ) { $version32 = $version32 -replace( $regex01, ".") }
   @{
     URL32    = $url32
+    FallbackURL32 = $fallbackUrl32
     Version  = Get-Version $version32
   }
 }

